server.js

```diff
@@ -14,8 +14,17 @@ var STATE_SERVER_ACK_TIMEOUT = Number(process.env.SCC_STATE_SERVER_ACK_TIMEOUT)
 var BROKER_SERVER_CONNECT_TIMEOUT = Number(process.env.SCC_BROKER_SERVER_CONNECT_TIMEOUT) || 10000;
 var BROKER_SERVER_ACK_TIMEOUT = Number(process.env.SCC_BROKER_SERVER_ACK_TIMEOUT) || 10000;
 var SECURE = !!argv.s || !!process.env.SCC_BROKER_SERVER_SECURE;
-var LOG_LEVEL = Number(argv.l) || Number(process.env.SCC_BROKER_SERVER_LOG_LEVEL) || 1;
 var RECONNECT_RANDOMNESS = 1000;
+/**
+ * Log levels:
+ * 3 - log everything
+ * 2 - warnings and errors
+ * 1 - errors only
+ * 0 - log nothing
+ */
+var LOG_LEVEL = (typeof argv.l !== 'undefined') ?
+  Number(argv.l) : (typeof process.env.SCC_BROKER_SERVER_LOG_LEVEL !== 'undefined') ?
+  Number(process.env.SCC_BROKER_SERVER_LOG_LEVEL) : 1;
```

Comment:
We should use `if-else` statement here. The double `?:?:` notation is a bit confusing or people like me ;p